examples/aggregator/proto/BUILD

```diff
@@ -44,3 +44,22 @@ cc_grpc_library(
     well_known_protos = True,
     deps = [":aggregator_cc_proto"],
 )
+
+proto_library(
```

Comment:
Is this required? The proto files are only used by the rust code on the server side, not by any clients.